modules/configuring-vsphere-connection-details.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-vSphere-connection-details_{context}"]
+= Configuring vSphere connection details
+
+[role="_abstract"]
+Update the following vSphere configuration settings:
+
+* vCenter network address
+* vCenter username
+* vCenter password
+* vCenter address
+* vSphere data center
+* vSphere datastore
+* Virtual machine of the cluster
+
+.Prerequisites
+* The {ai-full} has finished installing the cluster successfully.
+* The cluster is connected to link:https://console.redhat.com/.
+
+.Procedure
+. In the Administrator perspective, navigate to *Home > Overview*.
+. Under *Status*, click *vSphere connection*. This opens the *vSphere connection configuration* wizard.
+. In the *vCenter* field, enter the network address of the vSphere vCenter server. This can be either a domain name or an IP address. It appears in the vSphere web client URL; for example `https://[your_vCenter_address]/ui`. 
+. In the *Username* field, enter the base64-encoded version of your vSphere vCenter username.
+. In the *Password* field, enter the base64-encoded version of your vSphere vCenter password. This password will be stored in the kube-system namespace of the cluster. 
++
+[WARNING]
+====
+An incorrect vCenter username or password will make the cluster nodes unschedulable.
+====
++
+. In the *Datacenter* field, enter the name of the vSphere data center that contains the virtual machines used to back-up the cluster.
+. From the *Default data store* list, select the vSphere data store that is to store the persistent data volumes.
```

Comment:
It's not a list to choose from (not a dropdown). It's just a regular text-input field, so the user types the value there.

Reply:
Yes, I see now. At the time I think it was a dropdown, so that must have been a bug. Same with the next one.